tools/HelixQAC/README.md

```diff
@@ -20,7 +20,7 @@ with the compiler command being a command used to build an iceoryx file. If you
 
 an example command if you use GCC (command taked from the build log):
 
-    CCT_Generator.sh -c /usr/bin/c++  -Diceoryx_utils_EXPORTS -I/home/perforce/iceoryx/iceoryx_utils/include -I/home/perforce/iceoryx/iceoryx_utils/platform/linux/include  -O3 -DNDEBUG -fPIC   -W -Wall -Wextra -Wuninitialized -Wpedantic -Wstrict-aliasing -Wcast-align -Wno-noexcept-type -Wconversion -std=gnu++14 -o CMakeFiles/iceoryx_utils.dir/source/posix_wrapper/unix_domain_socket.cpp.o -c /home/perforce/iceoryx/iceoryx_utils/source/posix_wrapper/unix_domain_socket.cpp
+    CCT_Generator.sh -c /usr/bin/c++  -Diceoryx_hoofs_EXPORTS -I/home/perforce/iceoryx/iceoryx_hoofs/include -I/home/perforce/iceoryx/iceoryx_hoofs/platform/linux/include  -O3 -DNDEBUG -fPIC   -W -Wall -Wextra -Wuninitialized -Wpedantic -Wstrict-aliasing -Wcast-align -Wno-noexcept-type -Wconversion -std=gnu++14 -o CMakeFiles/iceoryx_hoofs.dir/source/posix_wrapper/unix_domain_socket.cpp.o -c /home/perforce/iceoryx/iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp
```

Comment:
Does this actually work, is this somewhere tested? I mean where is this `-Diceoryx_hoofs_EXPORTS` defined since I do not find the corresponding `CCT_Generator.sh` script in our repo I assume the script uses something from us where we define that compiler variable?!

Reply:
It was tested in #757 by @dkroenke @marthtz and @toniglandy1. IIUC these are compiler switches from CMake.

Reply:
@elfenpiff 
`-D<libname>_EXPORTS` is automatically generated by cmake when building a shared library. 

From https://cmake.org/cmake/help/v3.0/command/set_target_properties.html:
> DEFINE_SYMBOL sets the name of the preprocessor symbol defined when compiling sources in a shared library. If not set here then it is set to target_EXPORTS by default (with some substitutions if the target is not a valid C identifier). This is useful for headers to know whether they are being included from inside their library or outside to properly setup dllexport/dllimport decorations.